### ISSUE TITLE
Error Enum.EmptyError in text_wrap!

### ISSUE THREAD

**Issue by Hermanverschooten** [andrewtimberlake/elixir-pdf#24]
If you try this 
```elixir
{:ok, pdf} = Pdf.new
pdf |> Pdf.set_font("Helvetica", 8)
pdf |> Pdf.text_wrap!({100,100}, {200,200}, "")
```
this results in
```
** (Enum.EmptyError) empty error
    (pdf 0.5.0) lib/pdf.ex:490: Pdf.text_wrap!/4
```

**Comment by Hermanverschooten** [comment]
The same happens with `text_wrap`

**Comment by andrewtimberlake** [comment]
I’m sorry this took so long to fix. I was just bitten by this bug, which took a number of hours to fix because it was buried in a multi page document with large tables—now I wish I had dealt with this sooner 🤦‍♂️
This was fixed in 6cf0d93

**Comment by Hermanverschooten** [comment]
No worries, it got fixed now.
